gtk/src/default/gtk-3.20/_headerbar.scss

```diff
@@ -1,5 +1,5 @@
 // Color variables dependant on $headerbar_bg_color defined in _colors.scss
-$_button_bg_color: lighten($headerbar_bg_color, 6%);
+$_button_bg_color: lighten($inkstone, 2.5%);
```

Comment:
Hmmmm... I would not do it like this. Inkstone is too different from headerbar_bg_color
Better in _drawing here:
https://github.com/ubuntu/yaru/blob/master/gtk/src/default/gtk-3.20/_drawing.scss#L199
This line is too long anyways. Just lighten them up by 4% for light and dark version

Reply:
Like this:
$button_fill: if($c == $bg_color, image(lighten($c, 4%), image(lighten($c, 2%))) !global;

Reply:
It turns out that since dark variant is #4a4a4a, the closest change is increasing `headerbar_bg_color` by 1.5%, which is  #4d4a4a, while using 4% lighten in drawing makes #494646